bluebottle/payments/serializers.py

```diff
@@ -5,9 +5,11 @@
 
 
 class OrderPaymentActionSerializer(serializers.ModelSerializer):
+    data = serializers.JSONField(source='payload')
+
     class Meta:
         model = OrderPaymentAction
-        fields = ('type', 'method', 'url', 'payload')
+        fields = ('type', 'method', 'url', 'payload', 'data')
```

Comment:
I want to sent a proper JSON object here `data`, but want to keep it backwards compatible so also keep the `payload` object.